pkg/controller/resourcequota/resource_quota_controller_test.go

```diff
@@ -1137,8 +1137,6 @@ func expectSyncNotBlocked(fakeDiscoveryClient *fakeServerResources, workerLock *
 
 	workerLockAcquired := make(chan struct{})
 	go func() {
-		workerLock.Lock()
```

Comment:
the reason why I removed them : static-check complains for a blank critical zone here.

Reply:
maybe is it better for something dummy code between Lock and Unlock...

Reply:
This looks very intentional, I hope a test fails when you remove it like this :)

Reply:
hmm.... I got you . maybe the static-check pass but the unit-test failed. will dig into it. thank you